src/www/model/errors.ts

```diff
@@ -124,6 +126,8 @@ export const enum ErrorCode {
   NO_ADMIN_PERMISSIONS = 10,
   UNSUPPORTED_ROUTING_TABLE = 11,
   SYSTEM_MISCONFIGURED = 12,
+  // server connection failed, but successfully install the Outline services
+  SYSTEM_MISCONFIGURED_WITH_SUCCESSFUL_INSTALLATION = 13,
```

Comment:
I think this error is no longer needed, as the UI will be handling the error directly.